packages/webdriver/src/constants.js

```diff
@@ -88,3 +88,11 @@ export const DEFAULTS = {
         type: 'object'
     }
 }
+/**
+ * w3c allowed capabilities
+ */
+export const W3C_ALLOWED_CAPABILITIES = [
```

Comment:
there are a bunch of caps here that are not allowed capabilities, see here https://w3c.github.io/webdriver/#capabilities

Reply:
yup, just fixed 😄

Reply:
I still see invalid caps in there

Reply:
only to fix travis tests, they were being used in the existing webdriverio code.